src/base/bittorrent/session.h

```diff
@@ -255,6 +255,12 @@ namespace BitTorrent
         virtual void setSaveResumeDataInterval(int value) = 0;
         virtual int port() const = 0;
         virtual void setPort(int port) = 0;
+        virtual bool isSslPeersEnabled() const = 0;
+        virtual void setSslPeersEnabled(bool enabled) = 0;
+        virtual int sslPeersPort() const = 0;
+        virtual void setSslPeersPort(int port) = 0;
+        virtual Path sslPeersCertificatesDirectory() const = 0;
+        virtual void setSslPeersCertificatesDirectory(const Path &path) = 0;
```

Comment:
I feel `Peers` is redundant in all such names.

Reply:
I re-named most variables to remove "Peers". Except `sslPeersCertificatesDirectory`, where it seems correct to keep it.

Reply:
> Except `sslPeersCertificatesDirectory`, where it seems correct to keep it.

I don't think so.
If `sslCertificatesDirectory` looks like it has too broad a meaning, then I would use `Torrents` instead of `Peers`. (After all, we are talking about per-torrent certificates, aren't we?)